configs/retinanet/README.md

```diff
@@ -16,6 +16,7 @@ The highest accuracy object detectors to date are based on a two-stage approach
 
 |    Backbone     |  Style  | Lr schd | Mem (GB) | Inf time (fps) | box AP | Config | Download  |
 | :-------------: | :-----: | :-----: | :------: | :------------: | :----: | :------: | :--------: |
+|    R-18-FPN     | pytorch |   1x    |   8.0    |       8.7      |  32.4  | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/retinanet/retinanet_r18_fpn_1x_coco.py) | [model](https://download.openmmlab.com/mmdetection/v2.0/retinanet/retinanet_r18_fpn_1x_coco/retinanet_r18_fpn_1x_coco_20220311-d325f4ae.pth) &#124; [log](https://download.openmmlab.com/mmdetection/v2.0/retinanet/retinanet_r18_fpn_1x_coco/retinanet_r18_fpn_1x_coco_20220311_125714.log.json) |
```

Comment:
Is this the result of a single machine and single card?

Reply:
@hhaAndroid 
Yes, I just used one card to train this model.